### PR TITLE
chore: use prettier-plugin-pkg and yarn, prefer `prettierrc` or its defaults

### PR DESCRIPTION
**Pull request by JounQin**
`ts-node` is still required by `synckit` on development

tracked at https://github.com/un-ts/synckit/issues/86

---

Besides, it is such an amazing experience that I just found you've been using my `synckit` package for a while!

I found this fun fact and was going to simplify https://github.com/ota-meshi/eslint-plugin-json-schema-validator/blob/main/src/utils/http-client/sync-http.ts a bit, the wrapper `syncGet` is not needed.

### CODE REVIEW

File: .github/FUNDING.yml [hunk 1]
```diff
@@ -1 +1,5 @@
-github: ota-meshi
+# These are supported funding model platforms
+
+github: [ota-meshi, JounQin]
+open_collective: rxts
+patreon: 1stG
```
Comment (ota-meshi) [inline .github/FUNDING.yml:5]: Thank you for this PR!
Are you interested in maintaining this project together?
If so, I invite you to this project.

Comment (JounQin) [reply .github/FUNDING.yml:5]: Let's do it!

**Comment by ota-meshi** [comment]
> Besides, it is such an amazing experience that I just found you've been using my synckit package for a while!

Yeah! synckit is an awesome package!

File: tests/fixtures/.prettierrc.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (JounQin) [inline tests/fixtures/.prettierrc.json:1]: Too many test snapshots ti follow, skip for now.

File: package.json [hunk 3]
```diff
@@ -1,119 +1,114 @@
...
-    "license": "MIT",
-    "bugs": {
-        "url": "https://github.com/ota-meshi/eslint-plugin-json-schema-validator/issues"
-    },
-    "homepage": "https://ota-meshi.github.io/eslint-plugin-json-schema-validator/",
-    "peerDependencies": {
-        "eslint": ">=6.0.0"
-    },
-    "dependencies": {
-        "@pkgr/utils": "2.1.0",
```
Comment (JounQin) [inline package.json:59]: This is an undirect dependency, `synckit` is using it.

File: .github/FUNDING.yml (hunk 1 above)
Comment (ota-meshi) [reply .github/FUNDING.yml:5]: I will invite you to projects to check JSON(C) with ESLint!

**Review by ota-meshi — APPROVED** [review]
LGTM! Thank you!

File: .github/FUNDING.yml (hunk 1 above)
Comment (ota-meshi) [reply .github/FUNDING.yml:5]: I invited you to the project on GitHub and npm. Please let me know if anything is leaking.

**Pull request merged**